source/stylesheets/govuk_frontend_toolkit/_conditionals.scss

```diff
@@ -0,0 +1,91 @@
+// Media query helpers. These make producing IE layouts
+// super easy.
+
+// The base css you write should be for mobile. You can
+// then add desktop styles on top.
+//
+// Usage:
+//
+//   div.columns {
+//     border: 1px solid;
+//
+//     @include media(desktop){
+//       width: 30%;
+//       float: left;
+//     }
+//     @include ie-lte(8) {
+//       something to fix visual bugs in old IE
+//     }
+//     @include ie(6) {
+//       padding: 0;
+//     }
+//   }
+
+
+$is-ie: false !default;
+$mobile-ie6: true !default;
+$is-print: false !default;
+
+@mixin screen {
```

Comment:
These two mixins plus the `$is-print` variable have been added to this file. The rest of frontend toolkit is untouched.